### PR TITLE
Browser compatibility matrix, closes #48

### LINKED ISSUE #48: Add browser compatibility matrix
* ~~First version could be generating it manually using [browser-badge](https://github.com/substack/browser-badge)~~ This is very broken, look at what it generated using their example matrix!
foo

* Second version could be writing tests ~~using [Testling](https://ci.testling.com/) for true CI, which addresses~~ [testling is dead](https://github.com/substack/testling/issues/111), we need another solution here to address #6 also

Current known matrix:

- Best functionality
  - Chrome: Mac, Windows
  - Firefox: Mac, Windows
  - IE/Edge: Windows
- Janky functionality
  - Chrome: Android
  - Safari: Mac, iOS

### PR DESCRIPTION
**Pull request by divayprakash**

### CODE REVIEW

**Comment by divayprakash** [comment]
Check this out!

**Comment by shakeelmohamed** [comment]
@divayprakash this is a great start, I'd like something a bit more horizontal and with smaller icons.
See the w3schools chart for inspiration 
capturfiles_3

**Comment by divayprakash** [comment]
Could you specify an icon size? I was thinking 32px but 64px works as well.

**Comment by divayprakash** [comment]
64 px
chrome

32 px
chrome32

**Comment by shakeelmohamed** [comment]
Let's try 64px

**Comment by divayprakash** [comment]
On it.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ Create a bookmark with `javascript: (function () { window.location.href = 'https
 
 ![screenshot](img/screenshot.png)
 
+## Browser Support
+
+|Platform/Browser|![chrome](img/icons/chrome.png)|![safari](img/icons/safari.png)|![firefox](img/icons/firefox.png)|![edge](img/icons/edge.png)|![ie](img/icons/ie.png)|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|![windows](img/icons/windows.png)|Yes|-|Yes|Yes|Yes|
+|![mac](img/icons/mac.png)|Yes|No|Yes|-|-|
+|![ios](img/icons/ios.png)|-|No|-|-|-|
```
Comment (shakeelmohamed) [inline README.md]: Firefox & Chrome exist on iOS, we should say "No" for those boxes.

File: README.md [hunk 2]
```diff
@@ -35,6 +35,15 @@ Create a bookmark with `javascript: (function () { window.location.href = 'https
 
 ![screenshot](img/screenshot.png)
 
+## Browser Support
+
+|Platform/Browser|![chrome](img/icons/chrome.png)|![safari](img/icons/safari.png)|![firefox](img/icons/firefox.png)|![edge](img/icons/edge.png)|![ie](img/icons/ie.png)|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|![windows](img/icons/windows.png)|Yes|-|Yes|Yes|Yes|
+|![mac](img/icons/mac.png)|Yes|No|Yes|-|-|
```
Comment (shakeelmohamed) [inline README.md]: We support Safari on Mac

File: README.md [hunk 3]
```diff
@@ -35,6 +35,15 @@ Create a bookmark with `javascript: (function () { window.location.href = 'https
...
+|![ios](img/icons/ios.png)|-|No|-|-|-|
+|![android](img/icons/android.png)|No|-|No|-|-|
```
Comment (shakeelmohamed) [inline README.md]: Don't change it, but I think Chrome on Android actually works without autoplay 😆

**Comment by shakeelmohamed** [comment]
Thanks @divayprakash! For future reference, I don't get a notification for updates on a PR until you comment. Next time, please leave a comment after you push commits 👍

**Comment by divayprakash** [comment]
@shakeelmohamed Will do!

PS: Do we need to add the browser compatibility matrix to the website as
well?

**Comment by shakeelmohamed** [comment]
@divayprakash I don't think we need it on the website, I'd like to keep the UI as simple as possible. You can take #5 if you want something to do 😃

**Pull request merged**